stablehlo/dialect/StablehloOps.h

```diff
@@ -52,6 +52,18 @@ limitations under the License.
 namespace mlir {
 namespace stablehlo {
 
+struct StablehloDialectVersion : public mlir::DialectVersion {
+  StablehloDialectVersion() = default;
+  StablehloDialectVersion(uint32_t dialectVersion)
+      : dialectVersion(dialectVersion){};
+
+  // The current dialect version.
+  static constexpr uint32_t kCurrentDialectVersion = 1;
```

Comment:
how does this version differ than the other version in VHLO ?

Reply:
That's a good question. Given both are serialization related really and that one is updated as process already, that could resolve other process questions. I was thinking of having this be separate in case it may be slower changing or there are changes that's only related to bytecode but not there or vice versa ... but simpler is better and I'm struggling to come up with cases where it wouldn't be sufficient.